src/main/resources/templates/db_scripts/migration/pending_payment_cross_module.ftl

```diff
@@ -63,19 +63,26 @@ UPDATE ${myuniversity}_${mymodule}.budget
   WHERE id=sub.budget_id;
 
 -- update ledger fiscal year records
-UPDATE ${myuniversity}_${mymodule}.ledgerFy as ledger_fy
-  SET
-    jsonb = jsonb || jsonb_build_object('available', (jsonb->>'available')::decimal - sub.total,
-                                        'unavailable', (jsonb->>'unavailable')::decimal + sub.total)
-  FROM (SELECT ledger.id AS ledgerId, transactions.fiscalYearId AS fiscalYearId, sum((transactions.jsonb->>'amount')::decimal) AS total
-        FROM ${myuniversity}_${mymodule}.transaction AS transactions
-        LEFT JOIN ${myuniversity}_mod_invoice_storage.invoices AS invoices ON transactions.jsonb->>'sourceInvoiceId'=invoices.id::text
-        LEFT JOIN ${myuniversity}_${mymodule}.fund AS fund ON transactions.fromFundId = fund.id
-        LEFT JOIN ${myuniversity}_${mymodule}.ledger AS ledger ON fund.ledgerId = ledger.id
-        WHERE invoices.jsonb->>'status' = 'Approved'
-          AND transactions.jsonb->>'transactionType'='Pending payment'
-          AND NOT transactions.jsonb ? 'awaitingPayment'
-        GROUP BY ledger.id, transactions.fiscalYearId) AS sub
-  WHERE ledger_fy.fiscalYearId=sub.fiscalYearId AND ledger_fy.ledgerId=sub.ledgerId;
+DO $$
+  BEGIN
+    IF EXISTS (SELECT 1 FROM information_schema.tables WHERE table_schema='${myuniversity}_${mymodule}' AND table_name='ledgerfy')
```

Comment:
Amazon's RDS PostgreSQL 10 service features a database “master” user without full access to information_schema.
In general we cannot assume that we can access information_schema.
This `IF EXISTS` may fail for many sys-ops that use restricted database users for security when they run upgrades.

MODFISTO-122 "Remove ledgerFY totals calculation logic" (#150) removed the ledgerFy table, RMB will automatically delete it because of this entry in schema.json:
```
    {
      "tableName": "ledgerFY",
      "fromModuleVersion": "mod-finance-storage-6.0.0",
      "mode": "delete",
      "auditingTableName": "NOT_EXISTING_AUDITING_TABLE"
    }
```
Note the `"mode": "delete"` entry.
This removal has been shipped with mod-finance-storage 6.0.0.

The removal makes this ledgerFY upgrade code useless.
Please completely remove this ledgerFY upgrade code.